processor/aggregationprocessor/testdata/config.yaml

```diff
@@ -0,0 +1,24 @@
+receivers:
+    examplereceiver:
+
+processors:
+    aggregation:
+        metrics:
+            include:
+                match_type: strict
+                metric_names: [host/cpu/usage, from]
+            action: update
+            names: [cpu/usage_time, to]
```

Comment:
You raised a good point in chat the other day about this processor being more complex if we allow multiple filter options. 

The approach here of creating a dependency between the order of `metrics_names` & `names` is not super intuitive imo. I'd prefer having only one target `name` instead of `names`. I had originally suggested we use the include filter for consistency with the other processors, but I'm not 100% sure it makes sense here anymore. Maybe just `name` & `new_name` would be simpler? Note none of the other processors (including the [spanprocessor](https://github.com/open-telemetry/opentelemetry-collector/tree/master/processor/spanprocessor)) allow a simple rename like this.

You should create a Github issue with your proposal anyway and we can discuss with the Collector maintainers as I'm sure they'll have their own views.

For me, I think it would clearer to write this same config as:
```yaml
processors:
  aggregation:
    # rename "host/cpu/usage" to "cpu/usage_time"
    - include:
        match_type: strict
        names: [ host/cpu/usage ]
      action: update
      name: cpu/usage_time
    # rename "from" to "to"
    - include:
        match_type: strict
        names: [ from ]
      action: update
      name: to
```

Also FYI see https://github.com/open-telemetry/opentelemetry-collector/issues/1081

Reply:
We actually did have a discussion around this in our meeting today with Dave and Quentin, and we decided to not have multiple renames in one block, so as you said, it will be updated to a single value for field `name` instead of `names`. However, we did decide to keep the MatchProperties struct for now because the future of this processor should also support regex, and that struct seems to include regex as well. From the linked issue, it seems like there is already a discussion around changes for this field. Therefore, I think what I am going to do is sticking to the MatchProperties field for now, but only restricting 1 element in the field `names`. I am also going to keep an eye on that issue, so I can keep my code updated when the change actually happens.

In terms of creating a Github issue, Dave and Quentin reviewed my [rough draft](extensionExtraMapValue) today, and suggested some changes. One of the main ones suggested by Quentin is an all-inclusive configuration that would allow users to perform all needed operations to a metric with only one config block.

For example,
right now, if a user wants to rename a metric "old_name" to "new_name", and then rename the label "old_label" to "new_label", this user has to write 2 different config blocks:
`include:"
   metric_names: [old_name]
action: update
name: new_name`
and
`include:
  metric_names: [new_name]
label_actions:
  key: old_label
  action: update
  new_key: new_label
`  
After the change, it would just be one block:
`include:
  metric_names: [new_name]
action: update
name: new_name
label_actions:
  key: old_label
  action: update
  new_key: new_label
`

Reply:
I would be fine with a simple string field. One option would be to do that, but then leave open the option of making it more complex later on.